src/DirectionProvider.jsx

```diff
@@ -33,6 +34,9 @@ export default class DirectionProvider extends React.Component {
   }
 
   getChildContext() {
+    if (!this.props.direction) {
```

Comment:
I think this might break if `direction` changes from `null` to LTR or RTL, since `withDirection` on children might be subscribed to a direction broadcast further up the chain. Right?

This might be crazy, but we could wrap `DirectionProvider` in `withDirection`, and set the broadcast state to the direction from context if `this.props.direction` is null. Essentially treating `null` as a sort of "inherit".

Reply:
> Question: I'm not sure if I need to change withDirection.jsx too. Will its context reference become stale because it's already subscribed?

Just noticed this. I don't think we can update `withDirection` to handle this since context doesn't trigger a render (which is why we're using brcase to begin with). 😢

Reply:
@yzimet ^

Reply:
> This might be crazy, but we could wrap `DirectionProvider` in `withDirection`, and set the broadcast state to the direction from context if `this.props.direction` is null. Essentially treating `null` as a sort of "inherit".

@garrettberg I like your idea. I tried to implement this, but there is a collision on the `direction` prop name from the HOC. Want to pair on this later in the week?

Reply:
> Want to pair on this later in the week?

@yzimet Absolutely! 🎉